lib/core/navigation/router_config/router_config.dart

```diff
@@ -1,5 +1,5 @@
 import 'package:authentication_app/core/navigation/routes/routes_name.dart';
-import 'package:authentication_app/feature/login/presentation/views/login_page.dart';
+import 'package:authentication_app/feature/login/presentation/pages/login_page.dart';
```

Comment:
The way you handled routing here is required when we route in web apps.
  As we can access any page by providing the url of any page.
  That is why we provide [path] and **pathParameters**
  For Mobile app you don't need to do so
  Just use **path** and assign the **Route.constant**
  
  replace value you provided in **name** with **path**
  Also you can use **builder** instead of **pageBuilder**